.pipelines/azure-pipelines.yml

```diff
@@ -24,8 +24,8 @@ variables:
   # This is used in functional tests to make certain tests mandatory
   # in CI, but optional during local development. 
   SCITT_CI: 1
-
   ACR_REGISTRY: scittoss.azurecr.io
+  ACR_REGISTRY_RELEASE: confidentialledgeracrprod.azurecr.io
```

Comment:
If we are pushing release images to the existing ACL prod ACR, why can't we also push the main-branch images to the ACL staging ACR instead of maintaining a different registry?

As said above, pushing and pulling to a new SCITT ACR would require some minor changes and permissions setup. I'm fine if we prefer to use it, but I believe we should either always use ACL ACRs (in both staging and prod) or have separate SCITT ACRs (in both staging and prod).

Sorry, I know I'm a broken record here, but I just want to make sure we commit to either one or the other: if we start mixing ACL and SCITT ACR, it may get confusing.

Reply:
I am more incline on creating a new ACR for scitt prod but as we discussed, it can be quite challenging as of today. 
I will switch from `scittoss `to `confidentialledgerstaging `for now

Reply:
Agreed.
I just created a new service connection and switched the staging registry